Resources/Prototypes/Entities/Objects/Weapons/Melee/breaching_sledge.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: breaching sledge
+  parent: BaseItem
+  id: BreachingSledge
+  description: A sledgehammer heavily modified and re-calibrated for faster swings, better structural damage, but considerably worse heft against personnel. The red on the hammerhead is spraypaint. Or rust. Or blood.
+  components:
+  - type: Sprite
+    sprite: Impstation/Prospectors/Weapons/breaching_sledge.rsi
+    state: icon
+  - type: MeleeWeapon
+    attackRate: 3
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 2
+        Structural: 20
+    soundHit:
+      collection: MetalThud
+  - type: Wieldable
+  - type: IncreaseDamageOnWield
+    attackRate: 4
```

Comment:
You currently cannot modify a weapon's attack on wield. The `attackRate` under `IncreaseDamageOnWield` should be removed. I didn't make this change cause I didn't know if you wanted to change the base attack speed to match or not.

Reply:
No problem. I'm fine with removing it. I'll remove it and update it accordingly.